docs/Manual/Programs/Starter/Options.md

```diff
@@ -236,12 +238,24 @@ amongst each other.
 
 - `--starter.disable-ipv6=bool`
 
-if disabled, the starter will configure the `arangod` servers
+if disabled, the _Starter_ will configure the `arangod` servers
 to bind to address `0.0.0.0` (all IPv4 interfaces)
 instead of binding to `[::]` (all IPv4 and all IPv6 interfaces).
 
 This is useful when IPv6 has actively been disabled on your machine.
 
+- `--starter.unique-port-offsets=bool`
+
+If set to true, all port offsets (of slaves) will be made globally unique.
+By default (value is false), port offsets will be unique per slave address.
+
+- `--starter.debug-cluster=bool`
```

Comment:
IMO we should not document this function (we may even want to explicitly hide it) since it's only purpose is to debug cluster startup and has not gotten the amount of thought it should have.